aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -94,91 +96,102 @@ void ImagingResampleHorizontalConvolution8u4x(
     uint32_t* C10_RESTRICT lineOut1,
     uint32_t* C10_RESTRICT lineOut2,
     uint32_t* C10_RESTRICT lineOut3,
+    int64_t out_xsize,
     const uint32_t* C10_RESTRICT lineIn0,
     const uint32_t* C10_RESTRICT lineIn1,
     const uint32_t* C10_RESTRICT lineIn2,
     const uint32_t* C10_RESTRICT lineIn3,
-    int xsize,
-    int* xbounds,
-    int16_t* kk,
+    const int64_t* idx_ptr_xmin,
+    const int64_t* idx_ptr_size,
+    const int16_t* kk,
     int kmax,
-    int coefs_precision);
+    unsigned int coefs_precision);
 
 void ImagingResampleHorizontalConvolution8u(
     uint32_t* C10_RESTRICT lineOut,
+    int64_t out_xsize,
     const uint32_t* C10_RESTRICT lineIn,
-    int xsize,
-    int* xbounds,
-    int16_t* kk,
+    const int64_t* idx_ptr_xmin,
+    const int64_t* idx_ptr_size,
+    const int16_t* kk,
     int kmax,
-    int coefs_precision);
+    unsigned int coefs_precision);
 
 void ImagingResampleVerticalConvolution8u(
     uint32_t* C10_RESTRICT lineOut,
-    const uint32_t* C10_RESTRICT imIn,
-    int xmin,
-    int xmax,
-    int16_t* k,
-    int coefs_precision,
-    int xin);
+    const uint32_t* C10_RESTRICT lineIn,
+    int64_t xsize,
+    int64_t ids_min,
+    int64_t ids_size,
+    const int16_t* k,
+    unsigned int coefs_precision);
 
 void ImagingResampleHorizontal(
     const at::Tensor & unpacked_output,
     const at::Tensor & unpacked_input,
     int ksize,
     const std::vector<at::Tensor>& horiz_indices_weights,
     unsigned int horiz_weights_precision) {
+
+  // Interpolation horizontal pass: we compute x-axis (image width) interpolation outputs.
+
+  // Input data is stored as
+  //   input = [r[0], g[0], b[0], a[0], r[1], g[1], b[1], a[1], r[2], g[2], b[2], a[2], ...]
```

Comment:
That's only for RGBA images right? (not RGB + channels_last) ?

Reply:
Sorry I got confused and thought this PR was made on top of  #96848 but that\s the other way around - all good